web/source/text/prediction/modelManager.ts

```diff
@@ -358,6 +364,17 @@ namespace com.keyman.text.prediction {
 
     public set enabled(flag: boolean) {
       let keyman = com.keyman.singleton;
+
+      if(keyman.util.getIEVersion() == 10) {
```

Comment:
What does `getIEVersion()` return if it's not an IE browser? Should this be `===`?

Reply:
```
      // Finally test for IE 11 (and later?)
      n=agent.indexOf('Trident/');
      if(n < 0) {
        return 999;
      }
```

We're safe - it uses `999` for non-IEs.

Reply:
(kmwdevice.ts, around line 163 if you want to investigate more fully)

Reply:
Twitchy magic numbers but hey it works :)

Reply:
Twitchy magic numbers but hey it works :)